denizen_scripts/survival/repo-link/spawn_scripts/tutorial.dsc

```diff
@@ -221,4 +223,4 @@ tutorial_events:
     on player exits spawn_cuboid flagged:tutorial:
       - injet tutorial_skipped
```

Comment:
pls change `injet` to `inject` for Xeane his poor eyes

Reply:
Done